containers/python-examples/NLTK/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3.4"
+
+services:
+  grpc-api:
```

Comment:
We shouldn't be using `grpc-api` anymore. We should likely call this simply `server`. Ditto for all the other compose files.